lib/plugins/invoke/invoke.js

```diff
@@ -114,6 +119,13 @@ class Invoke {
 
     _.merge(process.env, defaultEnvVars);
 
+    // Turn zero or more --env options into an array
+    //   ...then split --env NAME=value and put into process.env.
+    Array.concat(this.options.env || []).forEach(itm => {
```

Comment:
Is it possible to use `lodash` library? We make a point of using it for consistency of code base when implementing if-check as possible as you can.